how-to-docs/drone-snippets.md

```diff
@@ -111,16 +117,15 @@ Create a file `sonar-project.properties` in the root of your project with the fo
 
 ```
 # Name of your git repo
-sonar.projectKey=rmp-prototype
+sonar.projectKey=your_repo
 # Name of your git repo
-sonar.projectName=rmp-prototype
+sonar.projectName=your_repo
 # Language
-#sonar.language=js
+#sonar.language=language
```

Comment:
In keeping with the way you've done parameters, above, this should be `<language>` (and likewise for the changed lines immediately above).